hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/QuotaObserverChore.java

```diff
@@ -764,6 +765,9 @@ int getNumRegions(TableName table) throws IOException {
       List<RegionInfo> regions = this.conn.getAdmin().getRegions(table);
       if (regions == null) {
         return 0;
+      } else {
```

Comment:
Nit: the `else` branch is unnecessary. Just make this `if (regions==null) {return 0;} RegionReplicaUtil...` (but properly formatted).

Reply:
Hi @joshelser , Thanks for the review. Did all the changes and pushed. :)